partials/specials/banners/support-video.php

```diff
@@ -1,21 +1,18 @@
 <?php
 // Get options and values
-$support_section_text = NM_get_option('nm_fundraising_settings_support_section_text', 'nm_fundraising_options');
-$heading_copy = !empty($args['heading_copy']) ? $args['heading_copy'] : NM_get_option('nm_fundraising_settings_support_section_title', 'nm_fundraising_options');
+$default_support_section_text = NM_get_option('nm_fundraising_settings_support_section_text', 'nm_fundraising_options');
+$default_support_section_heading = !empty($args['heading_copy']) ? $args['heading_copy'] : NM_get_option('nm_fundraising_settings_support_section_title', 'nm_fundraising_options');
 $customYoutube_id = NM_get_option('nm_fundraising_settings_video_banner_cta_youtube_id', 'nm_fundraising_options');
-$cutomText = NM_get_option('nm_fundraising_settings_video_banner_cta_text', 'nm_fundraising_options');
-$customHeadline = NM_get_option('nm_fundraising_settings_video_banner_cta_headline', 'nm_fundraising_options');
+$custom_text = NM_get_option('nm_fundraising_settings_video_banner_cta_custom_text', 'nm_fundraising_options');
+$custom_headline = NM_get_option('nm_fundraising_settings_video_banner_cta_headline', 'nm_fundraising_options');
 $youtube_id = get_post_meta($post_id, '_nm_support_youtube', true);
```

Comment:
There is never any `$post_id` declared from the Support page in any way. So I don't think this line could work. It also should have a more descriptive variable name